packages/contracts/contracts/recomender/Questionnaires.sol

```diff
@@ -22,9 +22,14 @@ contract Questionnaires is
 
     /// @notice Initializes the contract
     /// @dev Uses the initializer modifier to to ensure the contract is only initialized once
-    function initialize() public initializer {
+    function initialize(string[] calldata cids) public initializer {
         __AccessControl_init();
         _addDomain("snickerdoodle.com");
+
```

Comment:
Need to grant caller the DEFAULT_ADMIN_ROLE

Reply:
Oh nvm, Lucas's PR got it: 

https://github.com/SnickerdoodleLabs/protocol/pull/1220/files#diff-72cf84a6238904373cb18578a5382066b829b700d107b13e60ec4c5f19bd0668R29